crates/autopilot/src/infra/solvers/dto/settle.rs

```diff
@@ -11,6 +11,8 @@ pub struct Request {
     /// Unique ID of the solution (per driver competition), to settle.
     #[serde_as(as = "serde_with::DisplayFromStr")]
```

Comment:
why do we parse this as a `String`? should I also parse `block_deadline` as a String? :thinking:

Reply:
Isn't `DisplayFromStr` used everywhere because the value is surrounded by double quotes?